content/blog/2020/descriptors_in_the_wild.md

```diff
@@ -113,45 +111,43 @@ For Bitcoin Core (Alice):
 
 ```
 # new Extended wallet data
-export core_key=$(ku -n XTN -j create)
+export core_key=$(bdk-cli key generate)
 
 # New Extended Master Private
 
-export core_xprv=$(echo $core_key|jq -r '.wallet_key')
+export core_xprv=$(echo $core_key | jq -r '.xprv')
 
-# Derived Extended Pubblic key
+# Derived Extended Public key
 
-export core_xpub_84=$(ku -j -s 84H/0H/0H $core_xprv |jq -r '.public_version')
-export core_fingerprint=$(echo $core_key|jq -r '.fingerprint')
+export core_xpub_84=$(bdk-cli key derive --path m/84h/0h/0h --xprv $core_xprv | jq -r '.xpub' | rev | cut -c 3- | rev)
 
 # Now I build the xpubs (one for receiving and one for the change) 
 # together with informations about the derivation path to be communicated 
 # to BDK wallet's owner (Bob).
 
-export core_xpub_84_for_rec_desc="[$core_fingerprint/84'/0'/0']$core_xpub_84/0/*"
```

Comment:
To keep the example more or less the same you can use the `bdk-cli key derive` command to derive the pubkey for this descriptor